### ISSUE TITLE
Does it work on Windows? Or am I working in a wrong way?

### ISSUE THREAD

**Issue by EmilyJYN** [boredazfcuk/docker-icloudpd#222]
Hello to whoever reading this issue!

I am a newbie to docker (actually I learned docker by myself just for using this iCloud photo downloader) and am trying to run the code on a Windows 10 system. 

I worked on the "creating a container" part in README in this way: I saw the default download destination set as something like "/home/${user}/iCloud", which seems like a Linux destination. So I changed my download path to a Windows location like "--env download_path=A:\Learning\docker\icloud_download\docker_icloudpd\Photos", and volume to something like "--volume //A/Learning/docker/icloud_download/docker_icloudpd/Photos://A/Learning/docker/icloud_download/docker_icloudpd/Photos" "--volume //A/Learning/docker/icloud_download/docker_icloudpd/Photos/myname_config:/config". I thought this would then work on Windows. However, when I was trying to run the container, the program got stuck here:
![image](https://user-images.githubusercontent.com/52183621/196144672-2870c730-a38c-4af1-b3ab-1c2121680528.png)

Also, the "configuring a password" part in README says "run /usr/local/bin/sync-icloud.sh --Initialise", which is a location that doesn't exist on the Windows system. 

Now, to my limited knowledge, it seems like the container only works on Linux systems. Can someone please tell me if it does work on Windows in some way? Or am I trying this in a totally wrong way? I have been working on this for days and it doesn't work out, someone please help   TAT

**Comment by EmilyJYN** [comment]
Hi boredazfcuk!

First off, thanks a lot for your answer! I can now successfully use the docker container to download stuff to my virtual machine. 

However, I still want to move the photos onto my Windows system (the host). Thus, I tried to mount the virtual machine and the Windows by adding "--volume //A/Photos:/home/myname/iCloud" to my "docker create" command, where the location before the colon is on my Windows system, and the location after the colon is on my virtual machine. However, when I do have the photos on "/home/myname/iCloud" of my virtual machine, the photos did not appear on "A:\Photos" of Windows. 

I did some research on how to sync the files on the docker virtual machine and my Windows system, but didn't find anything useful. Finally, I am doing "docker cp container_name:/home/myname/iCloud A:\Photos" to copy the downloaded photos from the virtual machine to Windows. This method worked, but I am guessing that there should be a better way to use this docker container on Windows. 

Cheers,
